### PR TITLE
Sync up Foreman nightly tags build groups

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: configs/foreman/nightly.yaml [hunk 1]
```diff
@@ -570,6 +567,7 @@
       srpm-build:
         - bash
         - centos-release
+        - redhat-release-server
```
Comment (evgeni) [inline configs/foreman/nightly.yaml:570]: while I find seeing rh-rel-server weird in this list, here centos is not removed?

Comment (ehelms) [reply configs/foreman/nightly.yaml:570]: Yea, its a good callout. I'm just the messenger, don't throw :egg: :egg: :egg:

Comment (evgeni) [reply configs/foreman/nightly.yaml:570]: Easter is over anyways :rabbit:

**Review by evgeni — APPROVED** [review]
given that reflects the status quo: ack

but we shall clean this up.

**Pull request merged**